.travis.yml

```diff
@@ -28,6 +28,12 @@ rvm:
   - 2.3.0
   - ruby-head
 matrix:
+  include:
+    - rvm: jruby-9.0.4.0 # Latest compiled version in http://rubies.travis-ci.org
```

Comment:
I think it's just downloading it from S3, not compiling.  JRuby also provides its own binaries. Why @travis-ci doesn't use them is a mystery to me. Got any friends @travis-ci that can help us figure this out?  https://github.com/bf4/rails/commit/6a314d44c439b0a1ff702f1ec0871dfc0bc705da it appears 9.0.5.0 is now available https://github.com/travis-ci/travis-ci/issues/5733

I'm doing the incredibly crazy

```yaml
before_install:
  - '[ "$JRUBY_OPTS" != "" ] && export JRUBY_OPTS="--dev -J-Xmx1024M" && rvm mount -r http://rubies.travis-ci.org/ubuntu/12.04/x86_64/jruby-9.0.5.0.tar.gz --verify-downloads 1 --binary --ignore-gemsets && rvm use jruby-9.0.5.0 --binary --ignore-gemsets && gem install bundler --conservative'
matrix:
  include:
      jdk: oraclejdk8
      rvm: default
      env:
          - "JRUBY_OPTS='--dev -J-Xmx1024M'"
          - "GEM='ap'"
```